ASB SEP 2025 Security Patches integration

AlamIntel opened the pull request:
_Integrating_ Google Android Security Bulletin Patches

Test done: STS r42 TCs Passed except Except 2 TC
CVE_2025_22429 (Issue id:133999)
android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
(Issue ID:133685)

Tracked-On: OAM-133846

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

AlamIntel committed:
ASB SEP 2025 Security Patches integration

Integrating Google Android Security Bulletin Patches

Test done: STS r42 TCs Passed Except 2 TC
CVE_2025_22429 (Issue id:133999)
android.security.sts.KernelLtsTest#testRequiredKernelLts_WARN
(Issue ID:133685)

Tracked-On: OAM-133846
Signed-off-by: Alam, SahibeX <sahibex.alam@intel.com>

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

tanujtekriwal reviewed (commented):
LGTM

tanujtekriwal reviewed (approved):
LGTM

shyjumon-n reviewed (approved):
LGTM

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /android-ci-local/celadon_a15_master-merge/28
